test/e2e/components/header-navbar.js

```diff
@@ -0,0 +1,14 @@
+class HeaderNavbar {
```

Comment:
this component is not used in the `simple-send` spec, but is added in order to build the skeleton of the Page Object Model patterns defined in the proposal. It will be used in other specs and can be expanded according to the specific needs.
Also more components can be added following this example